OpenRA.Mods.Common/Widgets/Logic/MainMenuLogic.cs

```diff
@@ -200,21 +198,17 @@ public MainMenuLogic(Widget widget, World world)
 				if (currentNews != null)
 					DisplayNews(currentNews);
 
-				// Only query for new stories once per day
-				var cacheValid = currentNews != null && DateTime.Today.ToUniversalTime() <= Game.Settings.Game.NewsFetchedDate;
-				if (!cacheValid)
-					new Download(Game.Settings.Game.NewsUrl, cacheFile, e => { }, (e, c) => NewsDownloadComplete(e, c, cacheFile, currentNews));
-
 				var newsButton = newsBG.GetOrNull<DropDownButtonWidget>("NEWS_BUTTON");
+
 				if (newsButton != null)
 				{
-					newsButton.OnClick = () =>
-					{
-						newsButton.AttachPanel(newsPanel);
-						newsHighlighted = false;
-					};
+					// Only query for news once per day.
+					var cacheValid = currentNews != null && DateTime.Today.ToUniversalTime() <= Game.Settings.Game.NewsFetchedDate;
```

Comment:
While you are at it you can make use of `DateTime.UtcNow` instead. http://blogs.msdn.com/b/kirillosenkov/archive/2012/01/10/datetime-utcnow-is-generally-preferable-to-datetime-now.aspx